labs/coding-101-rest-basics-ga/8.md

```diff
@@ -1,12 +1,12 @@
-## Step 8. Getting the Network Devices
+## Step 8: Get the network devices
 
-Now, look at the Get Network Devices endpoint. A Network Device transports data to/from host devices or other network devices. A network device is typically a hub, switch, or router. It can be connected to the network by an Ethernet cable or wireless connection. Your goal is to find the network devices and display information about them.
+Now, look at the GET Network Devices API. A network device transports data to and from host devices or other network devices. A network device is typically a hub, switch, or router. It can be connected to the network by an Ethernet cable or wireless connection. Your goal is to find the network devices and display information about them.
 
-For details about the Get Network Device API, consult the [DNA Center API Docs](https://developer.cisco.com/site/dna-center-rest-api//).
+For details about the GET Network Devices API, consult the [DNA Center API Docs](https://developer.cisco.com/site/dna-center-rest-api/).
 
 1. Set up the API Call in Postman. Use the token to authorize a request to GET the network devices from the controller.
 
-	![](/posts/files/coding-101-rest-basics-ga/assets/images/postman6.png)<br/><br/>
+	![](assets/images/postman6.png)<br/><br/>
```

Comment:
Heh, if we get rid of /hosts and replace with network-device then this is that example... will keep thinking on this.

Reply:
Keep me posted.